packages/core/src/components/Onboarding/index.tsx

```diff
@@ -19,6 +19,7 @@ import { PickPrimary } from './pages/PickPrimary'
 import { Referral } from './pages/Referral'
 import { ReviewPrimary } from './pages/ReviewPrimary'
 import { Welcome } from './pages/Welcome'
+const namer = require('color-namer')
```

Comment:
Btw, looks like the ES6 `import` change didn't go through for some reason!

Reply:
Oh, Could be because the npm library does not provide any typings. I found a solution which I will deploy in the next commit. Let me know if that works.